sonar-xml-plugin/src/main/resources/org/sonar/l10n/xml/rules/xml/S105.json

```diff
@@ -11,5 +11,6 @@
   ],
   "defaultSeverity": "Minor",
   "ruleSpecification": "RSPEC-105",
-  "sqKey": "IllegalTabCheck"
+  "sqKey": "IllegalTabCheck",
```

Comment:
I feel this should be `S105` now, why not?

Reply:
It's like this because of the `IllegalTabCheck` legacy key of RSPEC-105. I want to keep the value in RSPEC to find the rule even with the previous key.